### ISSUE TITLE
Occluded object detection

### ISSUE THREAD

**Issue by jaideep11061982** [OpenGVLab/InternImage#2]
How good the model can be in detecting the occluded objects lying over each other from different categories ?

**Comment by jaideep11061982** [comment]
@OpenGVLab @whai362 @czczup @ZhenhangHuang

**Comment by yexiguafuqihao** [comment]
It's fantastic to work for the computer community. Would you please detail the training setup for the detector of object detection on the CrowdHuman dataset? What's the framework for occluded object detection? Is that Mask RCNN or Cascade RCNN? How to train the backbone of the detector to occluded object detection, Does the backbone need to be trained on a large-scale joint dataset? Does the detector need to be fine-tuned on Object365 first?

**Comment by wofmanaf** [comment]
Training configs of our xl model have been relesed in [crowd_human_config](https://github.com/OpenGVLab/InternImage/blob/master/detection/configs/crowd_human/cascade_internimage_xl_fpn_3x_crowd_human.py), the Cascade RCNN one, which is a little different from the paper.  [Backbone](https://github.com/OpenGVLab/InternImage/releases/download/cls_model/internimage_xl_22k_192to384.pth) is pre-trained on IN-22k and finetuned on IN-1k. We donot fine-tuning the detector on Object365.
